apps/src/dance/ai/constants.ts

```diff
@@ -1,6 +0,0 @@
-export const DANCE_AI_SOUNDS = [
```

Comment:
I see you added this file originally at https://github.com/code-dot-org/code-dot-org/pull/54871/commits/d7c102265b40bd292be7078c4492cc9fcf91ae7a. Just curious why at the time you had to add to a separate file and what has changed so now you can consolidate.

Reply:
Yep the only reason I created a separate file was because i wanted to make a TypeScript file in order to use the `as const` declaration. That's used to create the [`DanceAiSound` type](https://github.com/code-dot-org/code-dot-org/blob/a7b338e0dff16f4e5509742b1a95e903fc1ad6e3/apps/src/dance/ai/types.ts#L32) - the `as const` makes it so we can enforce that `DanceAiSound` is one of the constant values in `DANCE_AI_SOUNDS`. At the time, I opted to just create a new typescript file to not disrupt other existing files, but my preference would have been to put it in the same constants file. However, that's a not a strong preference - if we feel that keeping `dance/ai/constants` separate is better, then I'm fine with that too

Reply:
Gotcha - thanks!